### PR TITLE
Adds ability to share messages from client with py

### PR DESCRIPTION
**Pull request by urjitbhatia**
This PR fixes: https://github.com/sansyrox/robyn/issues/440

### CODE REVIEW

**Comment by urjitbhatia** [comment]
Note: This isn't a real performance regression since it changes the what the test is doing a bit.

File: integration_tests/base_routes.py [hunk 1]
```diff
@@ -17,20 +17,23 @@
...
-    return response
+    state = websocket_state[websocket_id]
+    if state == 0:
+        response["resp"] = "Whaaat??"
+    elif state == 1:
+        response["resp"] = "Whooo??"
+    elif state == 2:
+        response["resp"] = "*chika* *chika* Slim Shady."
+    websocket_state[websocket_id] = (state + 1) % 3
+    return json.dumps(response)
```
Comment (sansyrox) [inline integration_tests/base_routes.py:36]: we can use `jsonify` method from robyn here

**Comment by urjitbhatia** [comment]
@sansyrox switched to `jsonify` and also restored the original WS `benchmark` test since including the json parsing in it pollutes what is actually getting measured.

**Review by sansyrox — APPROVED** [review]
LGTM! Great work 🔥

**Pull request merged**